index.html

```diff
@@ -0,0 +1,404 @@
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="online_learning ">
+                        <div class="flex3">
+                            <div class="part1 ">
+                                <div class="flex3_1">
+                                    <div class="logo "><img src="img/home.jpg " alt=" "></div>
+                                    <div class="fignia ">
+                                        <p>Online learning</p>
+                                    </div>
+                                    <div class="number ">
+                                        <p>95%</p>
+                                    </div>
+
+                                    <div class="linegraphic "><img src="img/Vector (26).png " alt=" "></div>
+                                </div>
+                            </div>
+                            <div class="part2 ">
+                                <div class="header ">
+                                    <p>Online study:</p>
+                                </div>
+                                <div class="flex3_2">
+                                    <div><img src="img/line1.jpg " alt=" "></div>
+                                    <div class="percent ">
+                                        <p>95%</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="chart ">
+                    <div class="header ">
+                        <p>Class Statistics</p>
+                    </div>
+                    <div class="img ">
+                        <img src="img/chart.jpg " alt=" "></div>
+                </div>
+            </div>
+
+            <div id="rightside">
+
+                <div class="header ">
+                    <p>Notice Board</p>
+                    <span class="bell_background ">
+                    <img src="img/Ellipse 12.png " alt="background "></span>
+                    <span class="bell ">
+                     <img src="img/Vector (10).png " alt="bell "></span>
+                </div>
+
+                <div class="flex4">
+                    <div class="require ">
+                        <div class="notice_board ">
+                            <div><img src="img/pic1.png " alt=" "></div>
+                            <div class="text">
+                                <div class="text1 ">
+                                    <p>This page require data that you entered....</p>
+                                </div>
+                                <div class="text1_1">
+                                    <div class="text2 ">
+                                        <p>Wed,Feb 24,2021; </p>
+                                    </div>
+                                    <div class="text3 ">
```

Comment:
```suggestion
                                    <div class="text3">
```

why do you leave spaces in classnames?